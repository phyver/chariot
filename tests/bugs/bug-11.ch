data t('x) where C : 'x -> t('x)

data q where D : t(undefined) -> q

