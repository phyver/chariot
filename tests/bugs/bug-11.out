>> option verbose is set to 1
>> inductive type t was succesfully defined
** error: type undefined doesn't exist
